plugin-hrm-form/src/types/types.ts

```diff
@@ -59,11 +63,13 @@ export type CaseInfo = {
   referrals?: ReferralEntry[];
   incidents?: IncidentEntry[];
   documents?: DocumentEntry[];
+  caseSummaries?: CaseSummaryEntry[];
```

Comment:
Same as above, this should not be an array or optional, cases will always have 1 case summary,
```suggestion
  caseSummary: t.CaseSummaryEntry;
```

Reply:
Done!